modules/ROOT/pages/cloud-dataplane/overview/overview.adoc

```diff
@@ -0,0 +1,25 @@
+:page-layout: api-partial
+
+The Redpanda Cloud Data Plane APIs enable you to programmatically manage the resources within your clusters, including topics, users, access control lists (ACLs), and connectors. 
+
+To familiarize yourself with Redpanda Cloud API basics, see the xref:redpanda-cloud:manage:api/cloud-api-overview.adoc[Redpanda Cloud API Overview]. To get started using the API, try the xref:redpanda-cloud:manage:api/cloud-api-quickstart.adoc[Quickstart].
+
+NOTE: Redpanda Cloud uses a control plane and data plane architecture. To see the available endpoints for managing your clusters, networks, and resource groups, see the link:https://docs.redpanda.com/api/cloud-controlplane-api.html[Control Plane API Reference].
```

Comment:
@JakeSCahill I wasn't sure if I was supposed to use the .adoc extension for this link, so I kept it to .html just in case. Let me know if that should be changed